### DISCUSSION TITLE
Cannot read property 'elements' of null - Next JS

### DISCUSSION THREAD

**Discussion by ivanjeremic** [Gamote/lottie-react#26] [General]
Sometimes I get this: TypeError: Cannot read property 'elements' of null |   lottieRef.current.animationItem.renderer.elements[2];`

I have my ref and the code runs in `onDOMReady`, I did everything to prevent SSR, I dynamically import it and set `SSR` to `false`, on the initial page load everything is fine but when I navigate through the Site and then go back to the page where the Lottie animation is I get this error. 

Any Idea why this is happening,  I was sure the `onDOMLoaded` method is safe and works stable but it seems something is messing things up, please help. Thanks

**Comment by Gamote** [comment]
Hi @ivanjeremic

Please recreate the use case in [CodeSandbox](https://codesandbox.io/s) and I will have a look.
